build.gradle

```diff
@@ -28,6 +28,10 @@ repositories {
 dependencies {
   	  // https://mvnrepository.com/artifact/com.google.guava/guava
   	  compile group: 'com.google.guava', name: 'guava', version: '23.0'
+  	  // https://mvnrepository.com/artifact/com.itextpdf/itext7-core
+    compile group: 'com.itextpdf', name: 'itext7-core', version: '7.1.1'
```

Comment:
Ich hab hier ein Beispiel gefunden: https://developers.itextpdf.com/examples/tables/clone-101-very-simple-table

Ich weiss aber nicht wie leicht/schwer sich das in die bestehende Excel Funktionalitaet einbringen laesst

Reply:
Ich denke eher, dass wenn man es umsetzen will, es Sinn macht wenn man das PDF komplett neu erstellt. Man müsste den Header einfügen, dann oben die "statische" Tabelle mit den Werten und unten dann die dynamische Tabelle mit der Übersicht über die verkaufen Artikel anhand der Verkäufernummer. Der PDF Export macht ja nur Sinn für die Abrechnungen die man an die Leute rausgeben will. Würde mich schon reizen...